transforms/universal/c4_annotator/kfp_ray/c4_annotator_wf.py

```diff
@@ -0,0 +1,327 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+import os
+
+import kfp.compiler as compiler
+import kfp.components as comp
+import kfp.dsl as dsl
+
+from workflow_support.compile_utils import (
+    DEFAULT_KFP_COMPONENT_SPEC_PATH,
+    ONE_HOUR_SEC,
+    ONE_WEEK_SEC,
+    ComponentUtils,
+)
+
+# The secret name containing the s3 credentials.
+S3_SECRET = "s3-secret"  # pragma: allowlist secret
+
+EXEC_SCRIPT_NAME: str = "-m dpk_c4_annotator.ray.runtime"
+PREFIX: str = ""
+
+task_image = "quay.io/dataprep1/data-prep-kit/c4-annotator-ray:latest"
```

Comment:
I think this should be c4_annotator (Underscore not dash)
```suggestion
task_image = "quay.io/dataprep1/data-prep-kit/c4_annotator-ray:latest"
```